controller.go

```diff
@@ -272,6 +274,13 @@ func (c Controller) jsonHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	enc := json.NewEncoder(w)
+
+	clusterAckMsg := c.registry.clusterAck()
+	if clusterAckMsg != "" {
+		healthResults.Description = fmt.Sprintf("%s Cluster is acknowledged: %s", healthResults.Description, clusterAckMsg)
```

Comment:
Shouldn't you have: healthResults.Name instead of healthResults.Description as param to Sprintf, looks like you want to say that "%clustername% Cluster is acknowledged :question: